GameModes/Pathfinder/Pathfinder.csproj

```diff
@@ -1,4 +1,26 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
+    <Compile Update="**\*.axaml.cs">
+      <DependentUpon>%(Filename)</DependentUpon>
+    </Compile>
+    <AvaloniaResource Include="**\*.axaml">
+      <SubType>Designer</SubType>
+    </AvaloniaResource>
   </ItemGroup>
-</Project>
+  <ItemGroup>
+    <AvaloniaResource Remove="Views\PathfinderCharacterView.axaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Avalonia">
```

Comment:
Can you include these 2 package references and the ProjectReference in the Directory.Build.props for the GameModes folder? Every plugin will need them so i'd like to keep them in the same place.

Reply:
Yeah, I was thinking that too.  Have a sort of... plugin starter pack of references.

Reply:
Done!